packages/xstate-wallet/src/store/channel-store-entry.ts

```diff
@@ -325,7 +325,7 @@ export class ChannelStoreEntry {
 
   private static prepareStateVariables(
```

Comment:
This method should be removed -- state variables needed to be prepared because the stuff stores were objects that looked like `{_hex: '0x123'}`. They needed to be turned into `BigNumber.from('0x123')`.

Now, things that are stored look like `'0x123'`, and don't need to be prepared.